sys/include/net/ndp.h

```diff
@@ -157,7 +157,7 @@ extern "C" {
 /**
  * @brief   Default for DupAddrDetectTransmits
  * @see     [RFC 4862, section 5.1](https://tools.ietf.org/html/rfc4862#section-5.1)
- * @note    May not be greater than 7.
+ * @note    Must not be greater than 7.
```

Comment:
The reference doesn't name any max value, where does `7` come from - or did I miss that?

Reply:
Can we focus on the topic of this PR? This is about replacing "may not" with "must not"! In the meantime I will research why that is.

Reply:
This is in fact [GNRC-specific](https://github.com/RIOT-OS/RIOT/blob/master/sys/include/net/gnrc/netif/ipv6.h#L48-L55) (so a **follow-up** should add something like "[…] with @ref net_gnrc").